src/platform/imx8ulp/include/platform/drivers/idc.h

```diff
@@ -0,0 +1,28 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright 2019 NXP
```

Comment:
Update copyright year and author.

Reply:
@TangleZ even if you copy a file from another platform please add your name to the authors lists. You can keep mine if you want and also add a comment something like  based on ``` src/platform/imx8m/include/platform/drivers/idc.h```